package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "cql-exec-service",
-  "version": "3.0.0-alpha",
+  "version": "3.0.1",
```

Comment:
I like the idea of getting in the habit of bumping the version number of this repository, like you do here. Maybe we can do this every time we push a new code update?